cartographer_ros/src/cartographer_node_main.cc

```diff
@@ -243,8 +253,7 @@ void Node::AddImu(const int64 timestamp, const string& frame_id,
         LookupToTrackingTransformOrThrow(time, frame_id);
     CHECK(sensor_to_tracking.translation().norm() < 1e-5)
         << "The IMU is not colocated with the tracking frame. This makes it "
-           "hard "
-           "and inprecise to transform its linear accelaration into the "
+           "hard and inprecise to transform its linear accelaration into the "
```

Comment:
The IMU frame must be colocated with the tracking frame. Transforming linear accelaration into the tracking frame will otherwise be imprecise.